apis/logging/v1/clusterlogging_types.go

```diff
@@ -131,6 +131,7 @@ type KibanaSpec struct {
 	Tolerations  []v1.Toleration   `json:"tolerations,omitempty"`
 
 	// Number of instances to deploy for a Kibana deployment
+	// +optional
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Kibana Size",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:podCount"}
 	Replicas *int32 `json:"replicas"`
```

Comment:
Needs omitempty to avoid serializing "replicas: nil". The default json package only looks at struct tags, it's not aware of the k8s comments like // +optional
```suggestion
	Replicas *int32 `json:"replicas,omitempty"`
```